src/appengine/handlers/download.py

```diff
@@ -79,18 +82,13 @@ def handle_public_testcase(self, blob_info, testcase, fuzzer_binary_name):
     if 'restrict-view-commit' in issue.labels:
       return False
 
-    self._send_blob(
-        blob_info,
-        testcase.key.id(),
-        is_minimized=True,
-        fuzzer_binary_name=fuzzer_binary_name)
     return True
 
-  def get(self, resource):
+  @handler_flask.get(handler_flask.HTML)
```

Comment:
Why do we need to add HTML ?

Reply:
yes, here we should create another type such as handler.REDIRECT or something. Actually calling get handler is important to make request.get works.